include/yen/pgr_ksp.hpp

```diff
@@ -53,7 +53,7 @@ class Pgr_ksp :  public Pgr_messages {
          m_start(0),
          m_end(0),
          m_K(0),
-         m_heap_paths(false) {
+         m_heap_paths(false) { //! heap is for debuging purpose to check whether the algorithm is running well or not 
```

Comment:
This is a false statement

Reply:
Okay, so What is the significance of heap_paths from the user's perspective?

Reply:
First define user